src/app/pages/CreateWalletPage/index.tsx

```diff
@@ -57,7 +57,7 @@ export function CreateWalletPage(props: CreateWalletProps) {
         />
       )}
       <Grid gap="small" pad="small" columns={size === 'small' ? ['auto'] : ['2fr', '2fr']}>
-        <Box background="background-front">
+        <Box background="background-front" style={{ filter: showConfirmation ? 'blur(5px)' : 'none' }}>
```

Comment:
In Chrome, backdropFilter works [MnemonicValidation/index.tsx#L66](https://github.com/oasisprotocol/oasis-wallet-web/blob/e465051dd4b11a5d160f44f459cbace50c8665fb/src/app/components/MnemonicValidation/index.tsx#L66), so this extra filter will just double-blur mnemonic.

In Firefox this will only blur the mnemonic, because it is easiest to maintain bluring elements inside CreateWalletPage

Reply:
`backdrop-filter` is also supported by firefox. Do you have any reasoning why this doesn't work? I wouldn't like to differentiate the codebase between the browsers at this stage...

Reply:
Doesn't seem to be supported
- https://caniuse.com/css-backdrop-filter
- https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter
- My firefox screenshot

Reply:
I don't think this is very important for ux, so I'm also happy with old behaviour

Reply:
It seems FF has some performance hickups with the `backdrop-filter` implementation and is currently disabled, but it will be implemented some time in the future: https://bugzilla.mozilla.org/show_bug.cgi?id=1578503
Closing the issue.